formidable/models.py

```diff
@@ -88,6 +88,7 @@ class Meta:
     help_text = models.TextField(null=True, blank=True)
     multiple = models.BooleanField(default=False)
     order = models.IntegerField()
+    parameters = JSONField(null=True, blank=True, default={})
```

Comment:
Why do we need blank and null while there is a default value?

Reply:
It's a habit for the Django models. When you use it from the admin panel, you can't save a blank field if you haven't specified  `null=True, blank=True` even with a declared default value. Maybe it's an overkill here but I'd like to keep it.